docker/Dockerfile

```
@@ -26,11 +26,14 @@ COPY build_scripts/fixup-mirrors.sh /usr/local/sbin/fixup-mirrors
 
 # setup entrypoint, this will wrap commands with `linux32` with i686 images
 COPY build_scripts/install-entrypoint.sh build_scripts/update-system-packages.sh /build_scripts/
```

We need to copy in build_utils.sh here because install-entrypoint.sh calls update-system-packages.sh for `manylinux201(0|4)_i686`
```suggestion
COPY build_scripts/install-entrypoint.sh \
     build_scripts/update-system-packages.sh \
     build_scripts/build_utils.sh \
     /build_scripts/
```

---

That should resolve the build errors for manylinux+i686.
Can anyone reproduce the musllinux build error?